app/faucet/views.py

```diff
@@ -90,12 +92,12 @@ def save_faucet(request):
         }, status=400)
     fr = FaucetRequest.objects.create(
         fulfilled=False,
-        github_username=github_profile,
-        input_github_username=profile_handle,
+        github_username=request.user.username,
         github_meta=checkeduser,
```

Comment:
@owocki Do we need or want the GH payload here? We could eliminate the additional call to GH entirely if we don't need `github_meta`/`checkeduser`